.travis.yml

```diff
@@ -16,9 +16,12 @@ matrix:
   include:
     - { python: "3.7", dist: xenial, sudo: true }
 install:
-  - pip install pytest "typing$TYPING_VERSION" "pytest-cov>=2.5.1"
+  - pip install flake8 pytest "typing$TYPING_VERSION" "pytest-cov>=2.5.1"
   # mypy can't be installed on pypy
   - if [[ "${TRAVIS_PYTHON_VERSION}" != "pypy"* ]] ; then pip install mypy ; fi
+before_script:
+  # stop the build if there are Python syntax errors or undefined names
+  - if [[ "${TRAVIS_PYTHON_VERSION}" == "3.7" ]] ; then flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics ; fi
```

Comment:
* The difference between the __before_script__ section and the __script__ section is that when a test returns zero then Travis CI hides the output of that test under a twisty which does not visually clutter Travis report.
* Running flake8 on multiple versions of Python 3 will return similar results because they all share a common Abstract Syntax Tree.  No need to burn up CPUs on redundant tests. Python 3.7 was selected because it is the most restrictive with the addition of __async__ and __await__ as reserved words in the language.
* Putting the flake8 configuration in some separate file has two downsides:
    1. It removes the flexibility to have [multiple flake8 runs](https://github.com/vnpy/vnpy/blob/master/.travis.yml#L14): One for the showstopper issues and the other with __--exit-zero__.
    2. When our contributors want to understand what __flake8__ means, they will need to look for __.flake8__ (they won't find one) then __tox.ini  (they won't find one)and then finally read through __setup.cfg__ to see if there is a __[flake8]__ section.  _Explicit is better than implicit._
* The comment does not say what flake8 is (a linter) but explains what these command line parameters are asking flake8 to do for us.  flake8 can do tons of style validation (see the second pass in the vnpy example above) but we are not asking it to do any of that in this invocation.
* I think the current PR title more accurately describes what flake8 is doing in this PR.  We are NOT merely doing __flake8 .__ with no command line options.

If you think you can create another PR that more closely corresponds to your desires, please proceed.